### ISSUE TITLE
pretrained model of Snet

### ISSUE THREAD

**Issue by Dave19950** [qixuxiang/Pytorch_Lightweight_Network#2]
Can you give the pretrained model of the SNET which is different from shuffleNetV2.
